introduction.md

```diff
@@ -1 +1,8 @@
 Introduction to the Dojo!  So far I like the Git path the best, but I have my eye on the JavaScript path, too!
+
+I also have some deadly skills, primarily:
+* Git
+* Python
+* Naginata
+* Katana
+* Bo Staff
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?